data/presets/amenity/clinic/dialysis.json

```
@@ -0,0 +1,21 @@
+{
+    "icon": "fas-kidneys",
+    "geometry": [
+        "point",
+        "area"
+    ],
+    "tags": {
+        "amenity": "clinic",
+        "healthcare": "dialysis",
+        "healthcare:speciality": "dialysis"
+    },
```

I didn't find any documentation about `healthcare:speciality=dialysis` on the wiki. It actually [recommends](https://wiki.openstreetmap.org/wiki/Tag:healthcare%3Ddialysis) to use [`healthcare:speciality=nephrology`](https://wiki.openstreetmap.org/wiki/Tag:healthcare:speciality%3Dnephrology) in combination with `healthcare=dialysis`.

Also, maybe we could use the `addTags` approach for this preset, like the following?

```suggestion
    "tags": {
        "healthcare": "dialysis"
    },
    "addTags": {
        "amenity": "clinic",
        "healthcare": "dialysis",
        "healthcare:speciality": "nephrology"
    }
```